resources/wf_builder.rb

```diff
@@ -247,7 +259,9 @@
           platform_version: node['platform_version'],
         }
 
-        runner = Mixlib::ShellOut.new("delivery api post runners \
+        # TODO: Rework this to call the API directly, so we don't have delivery-cli formatting things.
+        runner = Mixlib::ShellOut.new("delivery --non-interactive --no-color \
```

Comment:
Here is an example of how you can "call the API directly": https://github.com/chef-cookbooks/expeditor/blob/master/libraries/helpers.rb#L40-L48. 

Here is an example of it in use. https://github.com/chef-cookbooks/expeditor/blob/0fd150f127404a56789f96955a8fa279c01a89dc/libraries/helpers.rb#L23

Faraday comes with Chef, and the token data is easy enough to find in various scenarios.